<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="IQCTab">
       <property name="focusPolicy">
        <enum>Qt::TabFocus</enum>
       </property>
       <attribute name="title">
        <string>IQC质检</string>
       </attribute>
       <widget class="QFrame" name="frame">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>395</width>
          <height>441</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="lb_title1">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>20</y>
           <width>120</width>
           <height>36</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>新宋体</family>
           <pointsize>-1</pointsize>
           <weight>9</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 22px &quot;新宋体&quot;;</string>
         </property>
         <property name="text">
          <string>DTU测试</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QPushButton" name="bt_start1">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>180</y>
           <width>160</width>
           <height>36</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>新宋体</family>
           <pointsize>-1</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(17, 160, 255);
font: 22px &quot;新宋体&quot;;</string>
         </property>
         <property name="text">
          <string>开始测试</string>
         </property>
        </widget>
        <widget class="QTextBrowser" name="textBrowser1">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>240</y>
           <width>321</width>
           <height>111</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>新宋体</family>
           <pointsize>-1</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 18px &quot;新宋体&quot;;</string>
         </property>
        </widget>
        <widget class="QLabel" name="lb_result1">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>380</y>
           <width>160</width>
           <height>36</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>新宋体</family>
           <pointsize>-1</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 22px &quot;新宋体&quot;;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="lb_describe1">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>80</y>
           <width>321</width>
           <height>81</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>新宋体</family>
           <pointsize>-1</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 18px &quot;新宋体&quot;;</string>
         </property>
         <property name="text">
          <string>测试前:
1.请连接好电脑与设备232接口
2.插卡及天线后，给设备上电
3.等待DTU灯亮起,点击开始测试</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
       <widget class="QFrame" name="frame_2">
        <property name="geometry">
         <rect>
          <x>395</x>
          <y>10</y>
          <width>395</width>
          <height>441</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="lb_result2">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>380</y>
           <width>160</width>
           <height>36</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>新宋体</family>
           <pointsize>-1</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 22px &quot;新宋体&quot;;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QTextBrowser" name="textBrowser2">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>240</y>
           <width>321</width>
           <height>111</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>新宋体</family>
           <pointsize>-1</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 18px &quot;新宋体&quot;;</string>
         </property>
        </widget>
        <widget class="QPushButton" name="bt_start2">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>180</y>
           <width>160</width>
           <height>36</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>新宋体</family>
           <pointsize>-1</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(17, 160, 255);
font: 22px &quot;新宋体&quot;;</string>
         </property>
         <property name="text">
          <string>开始测试</string>
         </property>
        </widget>
        <widget class="QLabel" name="lb_title2">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>20</y>
           <width>120</width>
           <height>36</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>新宋体</family>
           <pointsize>-1</pointsize>
           <weight>9</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 22px &quot;新宋体&quot;;</string>
         </property>
         <property name="text">
          <string>天线测试</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="lb_describe2">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>80</y>
           <width>321</width>
           <height>81</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>新宋体</family>
           <pointsize>-1</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 18px &quot;新宋体&quot;;</string>
         </property>
         <property name="text">
          <string>测试前:
1.请连接好电脑与设备232接口
2.插卡及天线后，给设备上电
3.等待DTU灯亮起,点击开始测试</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="Line" name="line">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>1</width>
           <height>441</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border-top: 1px solid red; background-color: rgb(0, 0, 0);</string>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(241, 241, 241);</string>
       </property>
       <attribute name="title">
        <string>整机质检</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextBrowser" name="textBrowser"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(241, 241, 241);</string>
       </property>
       <attribute name="title">
        <string>整机信号强度</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_2"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>TextLabel</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="textBrowser_2"/>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
